source/Mlos.Python/mlos/Grpc/BayesianOptimizerFactory.py

```diff
@@ -42,7 +42,7 @@ def create_remote_optimizer(self, optimization_problem: OptimizationProblem, opt
 
 
         Returns
-        =======
+        -------

```

Comment:
Are the fixups resulting from the WARNING output from the sphinx script?

Reply:
No, they are are results of me having had more coffee. We should probably fix the warnings. There are probably >100, we can error once we fixed them.